bullet_train-fields/Gemfile

```diff
@@ -8,6 +8,7 @@ gem "sqlite3"
 
 gem "sprockets-rails"
 
+gem "bullet_train", path: "../bullet_train"
```

Comment:
I can see this being helpful for tests for individual core gems, I remember the whole paths thing being an issue :+1:

Reply:
That's right, it goes back to tbis: https://github.com/bullet-train-co/bullet_train-core/pull/550#discussion_r1325760879